README.md

```diff
@@ -50,19 +50,21 @@ Getting started
    * **configure_running_julia_from_waf** -- Select "y" if and only if you intend to use Julia in your project and the Julia executable may be found on your path.
    * **configure_running_sphinx_from_waf** -- Select "y" if and only if you intend to use Sphinx in your project and the Sphinx executable may be found on your path.
    * **python_version** -- Usually accept the default. Must be a valid Python version >= 3.6
-   * **add_python_code_formatter_to_project** -- Choose yes if using Python
+   * **add_python_code_formatter_to_project** -- Choose yes if using Python.  <!--Implements black and other utilities as [pre-commit hooks]<https://pre-commit.com/>. For a full list of pre-commit hooks take a look at the [documentation]<http://hmgaudecker.github.io/econ-project-templates/>. -->
+   <!--* **add_pre_commit_linting** -- adds doc8, flake8, and restructuredtext-lint to the pre-commit hooks. -->
```

Comment:
Add explanation of what this does so that somebody who has no idea of what these packages are gets a sense of whether she wants it or not.

Reply:
"This" related to **add_pre_commit_linting** ... Thinking of it, maybe we should use

* **add_basic_pre_commit_hooks**
* **add_intrusive_pre_commit_hooks**

?